milestone.md

```diff
@@ -0,0 +1,199 @@
+2. Można utworzyć nową grę przez kliknięcie przycisku (+), przejście do ekranu [nowa gra](#nowa-gra)
+3. Przejście do ekranu [edycja gry](#edycja-gry) po klinięciu na daną grę.
+
+## [GM] Nowa gra / edycja gry
+Ekran do edycji i tworzenia gry wyglądają identycznie. Przy tworzeniu nowej gry wszystkie pola są domyślnie puste, a przy edycji zostają automatycznie wypełnione obecnymi informacjami.
+1. Wprowadzana jest nazwa gry
+2. Wprowadzany jest opis gry <br>
+3a. Wybierane jest czy gra ma startować automatycznie o zadanje godzinie (checkbox) <br>
+3b. Pokazuje się pole, w którym wprowadzana jest data i godzina startu
+4. Wybierane jest czy gra ma mieć automatyczne zakończenie. Jeżeli to potrzebne, pojawiają się pola z miejscem na wpisanie czasu końca lub wymaganej liczby punktów.
+5. Wyświetla się lista zadań <br>
+6a. Symbolem "+" można dodać nowe zadanie <br>
+6b. Klikając na istniejące zadanie można je edytować
+7. Użytkownik klika na przycisk "zapisz" (lub z odpowienim symbolem), który przesyła wprowadzone informacje na serwer
+8. Użytkownik może kliknąć na przycisk "rozpocznij oczekiwanie na graczy" (albo lepsza nazwa), który prześle do serwera informacje o zmianie stanu gry i przejście na ekran [oczekiwania na graczy](). Przed wykonaniem akcji wyświetla się popup z prośbą o potwierdzenie.
+
+## [GM] Nowe zadanie
+1. Pola tekstowe na tytuł i treść zadania
+2. Pole na wpisanie liczby punktów, może być wprowadzone za pomocą klawiatury lub zmienione przy pomocy przycisków + / -.
+3. Pole, które umożliwia zaznaczenie checkboxami jakie zadania należy wykonać zanim to zadanie stanie się dostępne.
+4. Przycisk (+) / "Dodaj pole odpowiedzi", po jego kliknięciu z listy rozwijanej wybierany jest typ TEXT, PHOTO, QR lub AUDIO.
+5. Przycisk "Zapisz"
+
+## [GM] Oczekiwanie na gracz
+1. Wyświetla się tytuł gry oraz lista graczy, którzy zgłosili się do danej gry
+2. Jeżeli gra ma czas automatycznego startu wyświetlane jest data startu
+3. Kliknięcie przycisku "start" przesyła do serwera informację o zmianie stanu i przechodzi na ekran [gra]().
+
+## [GM] Gra
+1. Wyświetla się nazwa gry
+2. Jeżeli jest ustalony czas zakończenia, wyświetla się pasek postępu
+3. Wyświetla się pasek postępu ile całkowicie zadań zostało wykonanych
+4. Przycisk "koniec gry", który kończy grę. Wymaga potwierdzenia przez popup. Po zakończeniu przechodzi do ekranu ranking.
+5. Wyświetla się liczba nieocenionych zadań, wraz z przyciskiem "oceniaj", który otwiera ekran [ocenianie](). 
+
+## [GM] Ocenianie 
+1. Co 5 sekund w tle pobierane są nowe odpowiedzi.
+2. Jeżeli wszystkie ocenione odpowiedzi są ocenione, wyświetla się stosowny komunikat.
+3. Jeżeli są nieocenione zadania, wybierane jest to najstarsze.
+4. Wyświetla się nazwa zadania, jego treść oraz przesłane subtaski.
+5. Zadanie można ocenić przy pomocy dwóch przycisków czerwony (odrzucenie) i zielony (akceptacja), bądź poprzez swipe jak na Tinderze.
+6. Po ocenianiu zadania, wyświetlane jest kolejne.
```

Comment:
Wracamy do recyclerview z nieocenionymi zadaniami?

Reply:
Uważam, że ten flow z recycler view będzie niergonomiczny. Przy sprawdzaniu dużej ilości odpowiedzi, GM będzie zapewne otwierał po kolei pierwsze z góry, co wygeneruje dużo upierdliwych kliknięć.

Reply:
Wydaje mi się 1, że to byłby 1 dodatkowy klik.